distributed/scheduler.py

```diff
@@ -136,8 +137,15 @@
     # {task key -> finish state}
     # Not to be confused with distributed.worker_state_machine.Recs
     Recs: TypeAlias = "dict[str, TaskStateState]"
+    # {client or worker address: [{op: <key>, ...}, ...]}
+    Msgs: TypeAlias = "dict[str, list[dict[str, Any]]]"
```

Comment:
Annoyingly, There doesn't seem to be any way to specify a TypedDict that *must* have a `op: str` pair and *may* have additional keys. Notably, jsonschema does allow for this.
Read https://stackoverflow.com/questions/71814658/python-typing-does-typeddict-allow-additional-extra-keys